cppcache/integration/test/PdxInstanceTest.cpp

```diff
@@ -395,9 +395,8 @@ TEST(PdxInstanceTest, testInstancePutAfterRestart) {
     cv_status.wait(lock, [&status] { return !status; });
   }
 
-  std::this_thread::sleep_for(std::chrono::seconds{30});
-
   for (auto& server : cluster.getServers()) {
+    server.wait();
```

Comment:
Okay I see this used separately here in several places, but... would it be necessary to call wait() in the other places if stop() just actually made sure the process was terminated?